main.py

```diff
@@ -24,7 +24,7 @@ def parse_args():
         help="String arguments for model, e.g. `pretrained=EleutherAI/pythia-160m,dtype=float32`",
     )
     parser.add_argument(
-        "--tasks", default=None  # , choices=utils.MultiChoice(sorted(ALL_TASKS))
+        "--tasks", default=None, choices=utils.MultiChoice(sorted(ALL_TASKS))
```

Comment:
Still need to find a way where this doesn't stop user from using `--include_path` because it errors out for tasks not in tasks.

Reply:
@haileyschoelkopf , what if we could print the list of `ALL_TASKS` if no task was matched?

Reply:
That could work I think! I uncommented this because it prevented anything being displayed when running `lm-eval -h`.

I think anything is fine so long as we raise an error immediately when any invalid task/group name is passed.

Reply:
I see your point, I've edit the help arg for tasks to display the available tasks. Let me know what you think.

Reply:
Just have the one comment re: raising a full error!